app/analytics/page.tsx

```
@@ -0,0 +1,383 @@
+              <div className="text-center">
+                <div className="text-2xl font-bold">{data.performance.summary.errorRate.toFixed(2)}%</div>
+                <div className="text-sm text-muted-foreground">Error Rate</div>
+              </div>
+            </div>
+
+            {data.performance.slowestOperations.length > 0 && (
+              <div>
+                <h4 className="font-semibold mb-3">Slowest Operations</h4>
+                <div className="space-y-2">
+                  {data.performance.slowestOperations.slice(0, 5).map((op, index) => (
+                    <div key={index} className="flex items-center justify-between p-2 bg-muted rounded">
```

Biome (1.9.4)

[error] Avoid using the index of an array as key property in an element.

This is the source of the key value.

The order of the items may change, and this also affects performances and component state.
Check the React documentation. 

(lint/suspicious/noArrayIndexKey)